proto/types/ibctypes.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package tendermint.spn.ibctypes;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/tendermint/spn/pkg/ibctypes";
+
+// MerkeRool represents a Merkel Root in ConsensusState
+message MerkeRool {
+    string hash = 1 [(gogoproto.moretags) = "yaml:\"hash\""];
```

Comment:
@froyd told it some slack thread:

> no, don’t use custom tags.. they’re not protojson compliant
> ...
> they should not be used. they’re there for backwards compatibility but when we will move to new proto codegen those will all disappear

Reply:
Ok good to know
Is `[(gogoproto.nullable) = false]` concerned as well?

Reply:
no, only the custom tags I think

Reply:
Based on the thread
https://github.com/tendermint/spn/issues/454